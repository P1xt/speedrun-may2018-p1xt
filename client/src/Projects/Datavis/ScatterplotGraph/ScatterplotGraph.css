.ScatterplotGraph {
  text-align: left;
  margin-top: 40px; }
  .ScatterplotGraph > h1 {
    text-transform: none; }
