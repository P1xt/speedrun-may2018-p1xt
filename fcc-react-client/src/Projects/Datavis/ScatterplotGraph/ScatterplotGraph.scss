.ScatterplotGraph {
  text-align: center;
  margin-top: 40px;

  & > h1 {
    text-transform: none;
  }

  .wrapper {
    padding: 40px 60px;
    margin: 20px auto;
  }

  svg {
    padding: 0 0 0 10px;
  }

  &_d3-tip {
    line-height: 1;
    font-weight: bold;
    padding: 12px;
    background: #000000;
    color: #fff;
  }

  circle {
    stroke: rgb(204, 204, 204);
    stroke-width: 1px;
    cursor: pointer;
  }

  circle:hover {
    stroke: black;
  }
  rect {
    fill: #222222;
  }

  .legend {
    fill: #fff;
  }
  .point-label {
    font-size: 10px;
  }
  .axis-label {
    fill: #222222;
    font-size: 1.1em;
    font-weight: bold;
  }
  .graph.heading {
    fill: #222222;
    font-size: 1.7em;
    font-weight: bold;
  }
  .graph.subheading {
    font-size: 1.4em;
  }
  .graph.subsubheading {
    font-size: 1em;
  }
}
