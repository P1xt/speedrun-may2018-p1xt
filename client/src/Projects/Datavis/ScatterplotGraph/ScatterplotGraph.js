import React from "react";
import { Container, Row, Col } from "reactstrap";

import "./ScatterplotGraph.css";

const ScatterplotGraph = () => {
  return (
    <div className="ScatterplotGraph">
      <Container>
        <Row>
          <Col>
            <h1>Scatterplot Graph</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default ScatterplotGraph;
