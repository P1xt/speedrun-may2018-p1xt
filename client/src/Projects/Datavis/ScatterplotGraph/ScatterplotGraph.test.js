import React from "react";
import renderer from 'react-test-renderer';
import { shallow } from "enzyme";

import ScatterplotGraph from "./ScatterplotGraph";

describe("<ScatterplotGraph />", () => {
  it("renders without crashing", () => {
    shallow(<ScatterplotGraph />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<ScatterplotGraph />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
