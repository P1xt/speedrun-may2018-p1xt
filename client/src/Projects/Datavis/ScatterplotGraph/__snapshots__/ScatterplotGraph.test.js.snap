// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ScatterplotGraph /> renders correctly 1`] = `
<div
  className="ScatterplotGraph"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          Scatterplot Graph
        </h1>
      </div>
    </div>
  </div>
</div>
`;
